astropy/table/table.py

```diff
@@ -2021,6 +2021,70 @@ def group_by(self, keys):
 
         return groups.table_group_by(self, keys)
 
+    def to_pandas(self):
+        """
+        Return a :class:`pandas.DataFrame` instance
+
+        Returns
+        -------
+        dataframe : :class:`pandas.DataFrame`
+            A pandas :class:`pandas.DataFrame` instance
+        """
+        from pandas import DataFrame
```

Comment:
Is there a general policy on doing try/except ImportError around optional dependencies?  I guess it would be a bit silly for someone to expect `to_pandas` to work without pandas being installed, but I wonder if it might be better to get a message like "pandas package must be installed to convert Table to DataFrame object".